pysc2/lib/renderer_human.py

```diff
@@ -102,6 +102,19 @@ def __init__(self, surf, surf_type, surf_rect, world_to_surf, draw):
     self.world_to_surf = world_to_surf
     self.draw = draw
 
+  def draw_arc(self, color, screen_loc, radius, scale, start_angle, stop_angle, thickness=1):
```

Comment:
Please make take the location and radius in world coordinates, then use self.world_to_surf to convert that to screen coordinates and distances. You shouldn't need to take/use scale at all.